Make some change(part 2) (#24)
kilacoda/chanim: README.md

# Chanim
This is an extension to [Manim](https://www.github.com/ManimCommunity/manim) library (initially created by [3Blue1Brown](https://github.com/3b1b/manim)),
for making videos regarding chemistry.

> > A Hindi version of this README is available [here](https://github.com/MsHeavyFour/chanim/blob/master/README-हिन्दी.md).

Reviewer: ```suggestion
> A Hindi version of this README is available [here](https://github.com/kilacoda/chanim/blob/master/README-हिन्दी.md).
```
link fix

> > A Chinese version of this README is available [here](https://github.com/MsHeavyFour/chanim/blob/master/README-自述文件.md).

Reviewer: ```suggestion
> A Chinese version of this README is available [here](https://github.com/kilacoda/chanim/blob/master/README-自述文件.md).
```

Explore the code and docs (coming soon!) for more on how to use chanim.

## Abilities

> Currently chanim only supports drawing compounds and reactions along with a few chemfig commands (Chemfig is a latex package that can draw chemical molecules such as coordination bonds. The description document website is [here](http://ctan.imsc.res.in/macros/generic/chemfig/chemfig-en.pdf)) but more is to come! If you have a suggestion, file an issue with a proper tag.

Reviewer: ```suggestion
Currently chanim only supports drawing compounds and reactions along with a few chemfig commands (Chemfig is a latex package that can draw chemical molecules and is used by chanim internally. The documentation is available [here](http://ctan.imsc.res.in/macros/generic/chemfig/chemfig-en.pdf)) but more is to come! If you have a suggestion, file an issue with a proper tag.
```
Fixed this a bit
